tests/hipscat_import/test_run_import.py

```diff
@@ -45,39 +45,39 @@ def test_dask_runner():
 
             runner.run_with_client(args, client)
 
-            # Check that the legacy metadata file exists, and contains correct object data
-            expected_lines = [
-                "{",
-                '    "cat_name": "small_sky",',
-                '    "ra_kw": "ra",',
-                '    "dec_kw": "dec",',
-                '    "id_kw": "id",',
-                '    "n_sources": 131,',
-                '    "pix_threshold": 1000000,',
-                r'    "urls": \[',
-                r'        ".*/small_sky_parts/catalog.*.csv"',
-                r'        ".*/small_sky_parts/catalog.*.csv"',
-                r'        ".*/small_sky_parts/catalog.*.csv"',
-                r'        ".*/small_sky_parts/catalog.*.csv"',
-                r'        ".*/small_sky_parts/catalog.*.csv"',
-                "    ],",
-                '    "hips": {',
-                r'        "0": \[',
-                "            11",
-                "        ]",
-                "    }",
-                "}",
-            ]
-            metadata_filename = os.path.join(args.catalog_path, "small_sky_meta.json")
-            ft.assert_text_file_matches(expected_lines, metadata_filename)
-
-            # Check that the catalog parquet file exists and contains correct object IDs
-            output_file = os.path.join(
-                args.catalog_path, "Norder0/Npix11", "catalog.parquet"
-            )
-
-            expected_ids = [*range(700, 831)]
-            ft.assert_parquet_file_ids(output_file, "id", expected_ids)
+        # Check that the legacy metadata file exists, and contains correct object data
+        expected_lines = [
+            "{",
+            '    "cat_name": "small_sky",',
+            '    "ra_kw": "ra",',
+            '    "dec_kw": "dec",',
+            '    "id_kw": "id",',
+            '    "n_sources": 131,',
+            '    "pix_threshold": 1000000,',
+            r'    "urls": \[',
+            r'        ".*/small_sky_parts/catalog.*.csv"',
+            r'        ".*/small_sky_parts/catalog.*.csv"',
+            r'        ".*/small_sky_parts/catalog.*.csv"',
+            r'        ".*/small_sky_parts/catalog.*.csv"',
+            r'        ".*/small_sky_parts/catalog.*.csv"',
+            "    ],",
+            '    "hips": {',
+            r'        "0": \[',
+            "            11",
+            "        ]",
+            "    }",
+            "}",
+        ]
+        metadata_filename = os.path.join(args.catalog_path, "small_sky_meta.json")
+        ft.assert_text_file_matches(expected_lines, metadata_filename)
+
+        # Check that the catalog parquet file exists and contains correct object IDs
+        output_file = os.path.join(
+            args.catalog_path, "Norder0/Npix11", "catalog.parquet"
+        )
+
+        expected_ids = [*range(700, 831)]
+        ft.assert_parquet_file_ids(output_file, "id", expected_ids)
 
 
 @pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
What's the reason we're ignoring this?

Reply:
It's happening somewhere inside dask, and there's nothing we can do about it.